develop/devguide/Connector/ConnectionsSnmpSnmpv3.md

```diff
@@ -0,0 +1,84 @@
+|[RFC 3418](https://datatracker.ietf.org/doc/html/rfc3418)     |Management Information Base (MIB) for the Simple Network Management Protocol (SNMP)         |
+|[RFC 3584](https://datatracker.ietf.org/doc/html/rfc3584)     |Coexistence between Version 1, Version 2, and Version 3 of the Internet-standard Network Management Framework         |
+
+# User-based Security Model (USM)
+
+SNMPv3 applies a User-based Security Model (USM) which provides support for authentication and encryption.
+For more information about USM in SNMPv3, refer to [RFC 3414](https://datatracker.ietf.org/doc/html/rfc2574).
+
+> [!NOTE]
+>
+> - Always verify that the correct security settings are filled in (username, authentication and privacy settings). From DataMiner 9.6.10 (RN 21458) onwards, a user is considered unique based on the following set of properties:
+>    - User name
+>    - Authentication protocol
+>    - Authentication password
+>    - Privacy protocol
+>    - Privacy password
+
+## Supported authentication and encryption protocols
+
+The following SNMPv3 authentication and encryption (privacy) protocols are supported in DataMiner:
+
+- Encryption algorithms:
+  - DES
+  - AES128
+  - AES192 (Since DataMiner 9.6.13 (RN 23586))
+  - AES256 (Since DataMiner 9.6.13 (RN 23586))
+- Authentication algorithms:
+  - MD5
+  - SHA128
+  - SHA224 (Since DataMiner 9.6.13 (RN 23586))
+  - SHA256 (Since DataMiner 9.6.13 (RN 23586))
+  - SHA384 (Since DataMiner 9.6.13 (RN 23586))
+  - SHA512 (Since DataMiner 9.6.13 (RN 23586))
+
+> [!NOTE]
+> When an encryption algorithm is used with more bits than the authentication type, this can cause issues. The following combinations are therefore not possible:
+>
+> - MD5 & AES192
+> - MD5 & AES256
+> - SHA128 & AES192
+> - SHA128 & AES256
+> - SHA224 & AES256
+
+## About snmpEngineID, snmpEngineBoots, snmpEngineTime and timeliness
+
+[RFC 3411](https://datatracker.ietf.org/doc/html/rfc3411#section-3.1.1.1) states the following:
+
+*Within an administrative domain, an snmpEngineID is the unique and unambiguous identifier of an SNMP engine. Since there is a one-to-one association between SNMP engines and SNMP entities, it also uniquely and unambiguously identifies the SNMP entity within that administrative domain. Note that it is possible for SNMP entities in different administrative domains to have the same value for snmpEngineID. Federation of administrative domains may necessitate assignment of new values.*
+
+Therefore, if DataMiner manages multiple SNMP entities, it is important that these all configured with a different value for *snmpEngineID*.
+If this is not the case, this can lead to communication issues:
+
+Every SNMP engine maintains two values:
+
+- *snmpEngineBoots*: a count of the number of times the SNMP engine has re-booted/re-initialized since *snmpEngineID* was last configured
```

Comment:
```suggestion
- *snmpEngineBoots*: a count of the number of times the SNMP engine has rebooted/reinitialized since *snmpEngineID* was last configured
```